config/crd/bases/rds.services.k8s.aws_dbclusterparametergroups.yaml

```diff
@@ -192,6 +153,41 @@ spec:
                   - type
                   type: object
                 type: array
+              parameterOverrideStatuses:
+                description: Provides a list of parameters for the DB cluster parameter
+                  group.
+                items:
+                  description: "This data type is used as a request parameter in the
```

Comment:
This misleading comment extracted automatically from https://github.com/aws/aws-sdk-go/blob/6cdd6178046217238d1969f5343fd929d7173703/service/rds/api.go#L44729